rts/Game/UI/SelectionKeyHandler.cpp

```diff
@@ -432,6 +432,26 @@ void CSelectionKeyHandler::DoSelection(std::string selectString)
 			return;
 		} break;
 
+		case hashString("SelectOneNearestMouse"): {
```

Comment:
My point is not against Mouse per se, just the terminology. One need to keep in mind this will be edited by players, `SelectClosestTo<Whatever>` sounds a bit more semantic to the selection keys structure

Reply:
I'm OK with whatever, so `SelectClosestToCursor` it is?